### DISCUSSION TITLE
Added a Raspberry Pi How-to

### DISCUSSION THREAD

**Discussion by yngves** [tillsteinbach/VWsFriend#440] [General]
Just wanted to mention that I've added instructions for [setting up a Raspberry Pi for VWsFriend](https://github.com/tillsteinbach/VWsFriend/wiki/Install) to the wiki. Hope someone finds it useful.

**Comment by MattiJarvinen** [comment]
This reminds me...

@tillsteinbach some fail2ban example for the wiki might be useful for some.

Although I personally just check for 401 amounts on nginx reverse proxy and ban with a HEAVY banhammer.

**Reply by tillsteinbach** [reply]
Hey @MattiJarvinen, unfortunatly I don't understand your point. Can you give me more information?

**Reply by MattiJarvinen** [reply]
Standard tooling for linux, reactive firewalls. Follow logs, find out harmful attempts via regex rules from certain IP addresses and block dynamically on iptables.

Most of us certainly run VWsFriend locally, so maybe this isn't really a VWsFriend thing anyway.

https://www.fail2ban.org/wiki/index.php/Main_Page
